etl/RetrosheetParser.py

```diff
@@ -24,14 +24,10 @@ def __init__(self):
                 , "padj": self.parse_padj
                 }
         
-        info_types = ['visteam', 'hometeam', 'site', 'date', 'number', 'starttime', 'daynight', 'usedh', 'umphome', 'ump1b', 'ump2b', 'ump3b', 'howscored', 'pitches', 'temp', 'winddir', 'windspeed', 'fieldcond', 'precip', 'sky', 'timeofgame', 'attendance', 'wp', 'lp', 'save', 'gwrbi']
```

Comment:
I removed this line because my recommendation is to store the game info data in a JSON format. The presence of the keys and the data types looks iffy and I don't think we have a compelling use for this data at the moment. If that changes we can go back to profile the data types and flatten the data as you propose.